app/controllers/pages_controller.rb

```diff
@@ -10,4 +10,16 @@ def meetup
 
   def contact
   end
+
+  def opportunities
+  end
+
+  def slack
+  end
+
+  def gs
+  end
+
+  def learn_ror
+  end
```

Comment:
It seems like we have 2 pages for learning resources.. lets combine them into one and just call it resources

Reply:
Sure thing! I’ll do that when I get in. 
What would you like in the navbar and what do you want in the footer?

Reply:
Could you go read a few blog posts about this and come back to me with an answer?

I feel like I want the main sections to be in the navbar, and the footer should be reserved for contact admin, join slack, find on GitHub etc. (Things that take a person away from the website).

This is probably something where after doing some research you can open up an issue describing what work needs to be done.